src/llm/ollama_client.py

```diff
@@ -22,7 +22,7 @@ def list_models():
 
     def inference(self, model_id: str, prompt: str) -> str:
         try:
-            response = ollama.generate(model=model_id, prompt=prompt.strip())
+            response = generate(model=model_id, prompt=prompt.strip())
```

Comment:
it should be:
response = client.generate(model=model_id, prompt=prompt.strip())

Reply:
Thank you. I've made the modification